themes/jb/layouts/sponsors/list.html

```diff
@@ -0,0 +1,25 @@
+{{ define "main" }}
+    <section class="spnosor-list px-4 mb-5">
+        <div class="container">
+            <div class="column is-full content">
+                <h1 class="title">Sponsors</h1>
+                <h4>We have many sponsors we love who support each show, and we hope you’ll enjoy them too:</h4>
+                {{range where .Site.Sections "Title" "Shows" -}}
+                    {{- range .Pages -}}
+                        {{- $show_sponsors := slice -}}
+                        {{- range (where (where .Pages "Type" "episode") "Date" "ge" (now.AddDate 0 0 (int $.Site.Params.Sponsors.lookbackdays) )) -}}
+                            {{- $show_sponsors = $show_sponsors | append .Params.Sponsors -}}
+                        {{- end -}}
+                        {{ if $show_sponsors -}}
+                            <h2>{{ .Title }}</h2>
+                        {{ end }}
```

Comment:
Nice job throwing in an if here for the show sponsors. If you, outside of the `if`, print the `$show_sponsors` you'll get an empty slice:
![image](https://user-images.githubusercontent.com/10230166/194727710-0e64f6d3-c212-490f-a1a1-66dec080d347.png)

which is the JB extras show: 
![image](https://user-images.githubusercontent.com/10230166/194727833-af7adba1-10e8-44e6-bf89-89248c314509.png)

The argument could be had to include the range inside the `if` statement so it doesn't try and evaluate it, but since it's a range and the slice is empty it's _technically_ fine.

Reply:
Moved into the if block just for safety.